node/hack/governor/src/index.ts

```diff
@@ -290,5 +295,71 @@ axios
     );
   })
   .catch((error) => {
-    console.error(error);
+    console.error("Request error:", error);
   });
+
+
+/*
+  Perform type checks on the incoming values
+  Check for a denylist set of characters
+
+  If either of these fail, we reject adding the token.
+
+  Example data: 30 000000000000000000000000b5c457ddb4ce3312a6c5a2b056a1652bd542a208 O404 omni404 18 1128.69 0xb5c457ddb4ce3312a6c5a2b056a1652bd542a208 7.4832146999999996
+*/
+async function safetyCheck(chain, wormholeAddr, symbol, coinGeckoId, tokenDecimals, tokenPrice, address, notional)  : Promise<boolean>{
+  
+  if(isNaN(chain)){
+    console.log("Invalid chain ID ", chain, " provided")
+    return false; 
+  }
+
+  if(await inputHasInvalidChars(wormholeAddr)){
+    console.log("Invalid wormhole address ", wormholeAddr, " provided")
+    return false; 
+  }
+
+  if(await inputHasInvalidChars(symbol)){
+    console.log("Invalid token symbol ", symbol, " provided")
+    return false; 
+  }
+
+  if(await inputHasInvalidChars(coinGeckoId)){
+    console.log("Invalid coin gecko id ", coinGeckoId, " provided")
+    return false; 
+  }
+
+  if(isNaN(tokenDecimals)){
+    console.log("Invalid token decimals ", tokenDecimals, " provided")
+    return false; 
+  }
+
+  if(isNaN(tokenPrice)){
+    console.log("Invalid token price ", tokenPrice, " provided")
+    return false; 
+  }
+
+  if(await inputHasInvalidChars(address)){
+    console.log("Invalid address ", address, " provided")
+    return false; 
+  }
+  if(isNaN(notional)){
+    console.log("Invalid notional", notional, " provided")
+    return false; 
+  }
+
+  return true; 
+}
+
+// Validates that a list of characters are not in the provided string.
```

Comment:
I think it actually does the opposite:
```suggestion
// Checks whether an illegal character is present in the provided string
```